### ISSUE TITLE
Feature request: Enable "Approve" button in Mist browser to pay without entering password each time

### ISSUE THREAD

**Issue by bamos01** [ethereum/mist#1121]
Similar to metamask- there are too many transactions when interacting with most dapps to enter your password each time.  Metamask allows you to click "approve" after the account is unlocked.  I realize there are some tradeoffs for security but it enables a much improved experience in many cases. 

#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files (Go to `menu -> accounts -> backup -> application data` and upload as zip-archive):
  -   osx: `~/Library/Application Support/Mist/node.log` 
  -   windows: `%APPDATA%/Roaming/Mist/node.log`
  -   linux: `~/.config/Mist/node.log`

**Comment by chevdor** [comment]
I agree with the idea. It also be can done like in geth where we unlock an account for N minutes.
Security is surely a concern but that could at least be enabled on testnet and save a bunch of keyboards 😄 

If we keep the security aspect in mind, the risk here is that users will not want to enter a password every time and despite the recommendations will setup weak passwords.

I would be fine as well wit the ability to benefit from Keychain but this is OS dependant.

**Comment by bamos01** [comment]
Note too - many transactions in dapps dont involve any value except the transaction fee
